site/content/en/docs/installation/_index.md

```diff
@@ -51,11 +51,15 @@ The webhook server in kueue uses an internal cert management for provisioning ce
 ## Install a released version
 
 To install a released version of Kueue in your cluster, run the following command:
-
 ```shell
 kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/{{< param "version" >}}/manifests.yaml
 ```
 
+To wait for Kueue to be fully available, run:
+```shell
+kubectl wait deploy/kueue-controller-manager -nkueue-system --for=condition=available --timeout=5m
```

Comment:
lgtm, qq @tenzen-y would you be in favor of adding this note also to the main page https://github.com/kubernetes-sigs/kueue?tab=readme-ov-file#installation? Or you find it too detailed?

Reply:
As far as I understand, the Deployment available status doesn't indicate the ready, right?
Even if the deployment status says available, the Pod container still initializing, right?

Could you check the behavior?

Reply:
Instead of this command, I would propose using the following command:

```shell
kubectl wait deploy/kueue-controller-manager -n kueue-system --for=jsonpath=.status.readyReplicas=1 --timeout=5m
```

As far as I know, the `readyReplicas` will be counted up when the readiness check succeeds.

Reply:
This would be an alternative approach, but I believe the original proposal also works, as I tested it many times, and it is shorted. Let me see if I can find some docs to support it.

Reply:
It should work fine. We already have readyz check that waiting for webhooks

https://github.com/kubernetes-sigs/kueue/blob/7951ff39bc6a4fcb23a48431b2f809b7055bf050/cmd/kueue/main.go#L308-L318

And deployment should wait for readyz check (https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L530-L532).

And also we have the same test here 

https://github.com/kubernetes-sigs/kueue/blob/7951ff39bc6a4fcb23a48431b2f809b7055bf050/test/util/e2e.go#L93-L96

that it's working fine on e2e test.

Reply:
When the cert controller take a long time to set up certs, I guess that the original approach often doesn't work well, right?

Reply:
It works in this situation too, unless it takes longer than 5min. As explained above, Deployment will wait with setting the Available condition until all replicas are ready. So essentially it is the same condition, but expressed shorter.

Reply:
Uhm that makes sense. So, the `conditions=Available` indicates the same situations as `.status.readyReplicas=1`.
If so, I'm fine with the original command.

> lgtm, qq @tenzen-y would you be in favor of adding this note also to the main page https://github.com/kubernetes-sigs/kueue?tab=readme-ov-file#installation? Or you find it too detailed?

We want to keep the Readme simple. So, it would be better to locate this guide only in the docs.

Reply:
sgtm

Reply:
Actually, there is a slight difference, let me summarize:

Reply:
Potentially different, but since Kueue does not set MinReadySeconds for its Deployement then it defaults to 0. So no difference after all: https://github.com/kubernetes/kubernetes/blob/60c4c2b2521fb454ce69dee737e3eb91a25e0535/pkg/apis/apps/types.go#L225-L229

Reply:
Maybe, you want to indicate https://github.com/kubernetes/kubernetes/blob/60c4c2b2521fb454ce69dee737e3eb91a25e0535/staging/src/k8s.io/api/apps/v1/types.go#L485-L524 instead of the StatefulSet one, right?

Anyway, that sounds good. Thank you for investigating!

Reply:
oh yes, this is what I meant, but it doesn't change the conclusion here.